src/app.js

```diff
@@ -14,6 +17,9 @@ const morganOption = (NODE_ENV === 'production')
 app.use(morgan(morganOption))
 app.use(helmet())
 app.use(cors())
+app.use('/api/auth', authRouter)
+app.use('/api/users', usersRouter)
+app.use('/company', companyRouter)
```

Comment:
(Optional) do we want to keep all routes /api/{variable} or just the user ones?

Reply:
All routes are necessary.

Reply:
change has been made